### DISCUSSION TITLE
:woman_cook: Integrate Emoji Kitchen Stickers

### DISCUSSION THREAD

**Discussion by NatVIII** [FelipeFTN/Emoji-Copy#24] [Ideas]
**Context**
[Emoji Kitchen](https://emojikitchen.dev/) is a project by Google that allowed for combinations of emojis typed in on GBoard to be pasted as an image or sticker. One dev who worked on the project writes about it [here](https://jenniferdaniel.substack.com/p/introducing-emoji-kitchen-). While I don't think there's any explicit license given to it, if it's content is datamined and added to an extension like this the chances of crackdown are unlikely.  

**Desired Goal**
Integrating Emoji Kitchen into emoji-copy, pasting the generated emoji images into the clipboard

**Initial Concerns**
Would this represent feature creep for emoji-copy that should be discouraged? How would this affect the UI? What should be the prominence of such an abnormal feature? 

This feature will require a lot of work and is probably a medium to long term project

**Discussion**
Since this is a hefty project many other features may have bigger priority, but feel free to discuss whether you think this is a good or bad idea; or how it can best be executed! I personally think it would be an extremely fun and charming addition to the Linux Desktop 🩵🩷🤍🩷🩵

**Comment by FelipeFTN** [comment]
Niice suggestion, @NatVIII! :tada: 
Taking a look at the Emoji Kitchen, it looks like their code is open source, so it's a lot better to learn how it works, and how it was build.
Also, the merged emojis are PNG images, it is getting from gstatic, which is a Google web domain for Google's static stuff, such as CSS, JS, images etc. 
Here is an example of gstatic emoji URL: https://www.gstatic.com/android/keyboard/emojikitchen/20230301/u1f606/u1f606_u1f606.png
We may need a way to search into gstatic for mashups emojis. :eyes: 
Despite that, maybe a new tab in emoji-copy may be created to support this new feature!
But stills a nice idea, let's move it to discussions and keep it open there, we shall be working on this feature idea! :bulb:

**Comment by finick-wilder** [comment]
Love how 👩‍🍳 Emoji Kitchen lets you cook up fun and quirky [sticker combos!](https://emojikitchen.tr/emoji-kitchen-combination/) 🍓+🐱 or 🤖+❤️—the possibilities are endless! It’s such a creative way to express moods and mash up feelings in chats. 🔥 Perfect for spicing up convos with personalized flair! 🧁🎨 #EmojiKitchen #StickerChef
